bin/configs/kotlin-spring-boot-delegate-nodefaults.yaml

```diff
@@ -0,0 +1,14 @@
+generatorName: kotlin-spring
+outputDir: samples/server/petstore/kotlin-springboot-delegate-nodefaults
```

Comment:
i ran `gradle build` in this folder but got
```
wing3@MSI MINGW64 ~/Code/openapi-generator/samples/server/petstore/kotlin-springboot-delegate-nodefaults (pstorch-kotlin_delegate_nodefaults_19211)
$ gradle build
Starting a Gradle Daemon (subsequent builds will be faster)

> Task :compileKotlin
e: C:\Users\wing3\Code\openapi-generator\samples\server\petstore\kotlin-springboot-delegate-nodefaults\src\main\kotlin\org\openapitools\api\PetApiController.kt: (10, 9): Platform declaration clash: The following declarations have the same JVM signature (getDelegate()Lorg/openapitools/api/PetApiDelegate;):
    fun `<get-delegate>`(): PetApiDelegate defined in org.openapitools.api.PetApiController
    fun getDelegate(): PetApiDelegate defined in org.openapitools.api.PetApiController
e: C:\Users\wing3\Code\openapi-generator\samples\server\petstore\kotlin-springboot-delegate-nodefaults\src\main\kotlin\org\openapitools\api\PetApiController.kt: (14, 5): Platform declaration clash: The following declarations have the same JVM signature (getDelegate()Lorg/openapitools/api/PetApiDelegate;):
    fun `<get-delegate>`(): PetApiDelegate defined in org.openapitools.api.PetApiController
    fun getDelegate(): PetApiDelegate defined in org.openapitools.api.PetApiController
e: C:\Users\wing3\Code\openapi-generator\samples\server\petstore\kotlin-springboot-delegate-nodefaults\src\main\kotlin\org\openapitools\api\StoreApiController.kt: (10, 9): Platform declaration clash: The following declarations have the same JVM signature (getDelegate()Lorg/openapitools/api/StoreApiDelegate;):
    fun `<get-delegate>`(): StoreApiDelegate defined in org.openapitools.api.StoreApiController
    fun getDelegate(): StoreApiDelegate defined in org.openapitools.api.StoreApiController
e: C:\Users\wing3\Code\openapi-generator\samples\server\petstore\kotlin-springboot-delegate-nodefaults\src\main\kotlin\org\openapitools\api\StoreApiController.kt: (14, 5): Platform declaration clash: The following declarations have the same JVM signature (getDelegate()Lorg/openapitools/api/StoreApiDelegate;):
    fun `<get-delegate>`(): StoreApiDelegate defined in org.openapitools.api.StoreApiController
    fun getDelegate(): StoreApiDelegate defined in org.openapitools.api.StoreApiController
e: C:\Users\wing3\Code\openapi-generator\samples\server\petstore\kotlin-springboot-delegate-nodefaults\src\main\kotlin\org\openapitools\api\UserApiController.kt: (10, 9): Platform declaration clash: The following declarations have the same JVM signature (getDelegate()Lorg/openapitools/api/UserApiDelegate;):
    fun `<get-delegate>`(): UserApiDelegate defined in org.openapitools.api.UserApiController
    fun getDelegate(): UserApiDelegate defined in org.openapitools.api.UserApiController
e: C:\Users\wing3\Code\openapi-generator\samples\server\petstore\kotlin-springboot-delegate-nodefaults\src\main\kotlin\org\openapitools\api\UserApiController.kt: (14, 5): Platform declaration clash: The following declarations have the same JVM signature (getDelegate()Lorg/openapitools/api/UserApiDelegate;):
    fun `<get-delegate>`(): UserApiDelegate defined in org.openapitools.api.UserApiController
    fun getDelegate(): UserApiDelegate defined in org.openapitools.api.UserApiController

```

can you please take a look when you've time?

can you also add the new folder to the workflow file: https://github.com/OpenAPITools/openapi-generator/blob/master/.github/workflows/samples-kotlin-server.yaml#L34 ?

Reply:
Thanks for the feedback. I've fixed it.